app/api/companies.py

```diff
@@ -278,3 +285,34 @@ def delete_company(id):
 
     cache.clear()
     return response
+
+
+@bp.route('/v1/upload', methods=['POST'])
+@jwt_required()
+def upload_file():
+    if 'file' not in request.files:
+        return error_response(400, "No file key in request.files")
+
+    img = request.files['file']
+    if img.filename == "":
+        return error_response(400, "Please select a file")
+
+    filename = secure_filename(img.filename)
+    file_ext = os.path.splitext(filename)[1]
+
+    if file_ext not in Config.UPLOAD_EXTENSIONS or file_ext != validate_image(img.stream):
+        return error_response(400, f"Invalid file extension, please use .gif, .jpg or .png")
+
+    output = {}
+    output['url'] = str(upload_file_to_s3(img, Config.S3_BUCKET_NAME))
+
+    response = jsonify(output)
+    response.status_code = 201
+
+    return response
+
+    # TODO: We could also think about nameing the file ourselves...
```

Comment:
This seems like a good idea 👍

But then you'd need the rest of the company data (the id and the name etc)

Reply:
I will include this in the issue #22, while also including Pillow to automatically resize the uploaded images.